diff --git a/node_modules/tape/lib/default_stream.js b/node_modules/tape/lib/default_stream.js
index d1a718a..1a1617e 100644
--- a/node_modules/tape/lib/default_stream.js
+++ b/node_modules/tape/lib/default_stream.js
@@ -1,5 +1,4 @@
 var through = require('through');
-var fs = require('fs');
 
 module.exports = function () {
     var line = '';
@@ -18,7 +17,7 @@ module.exports = function () {
     }
 
     function flush() {
-        if (fs.writeSync && /^win/.test(process.platform)) {
+        if (false) {
             try { fs.writeSync(1, line + '\n'); }
             catch (e) { stream.emit('error', e) }
         } else {
diff --git a/node_modules/tape/lib/test.js b/node_modules/tape/lib/test.js
index e58e33d..cdd64ee 100644
--- a/node_modules/tape/lib/test.js
+++ b/node_modules/tape/lib/test.js
@@ -1,6 +1,6 @@
 var deepEqual = require('deep-equal');
 var defined = require('defined');
-var path = require('path');
+var path = { sep: '/' };
 var inherits = require('inherits');
 var EventEmitter = require('events').EventEmitter;
 var has = require('has');
